applications/DEM_application/README.md

```diff
@@ -0,0 +1,121 @@
+
+In order to represent irregular particles with spheres, a numerical correction is used. the rolling friction imposes a virtual moment opposite to particle rotation and dependent on its size.
+
+#### Continuum materials Strategy
+
+For continuum materials simulations, the contact between particles can resist tractions up to a certain limit, when breakage occurs. Depending on the chosen constitutive law, the computation of the forces changes. In the basic versions, a bond strategy is used, but more advanced laws use a non-local stress-tensor based strategy.
+
+### DEM integration schemes
+
+The standard translational and rotational equations for the motion of rigid bodies are used to compute the dynamics of the spheres and clusters. The following schemes can be chosen separately for translation and rotation:
+
+* Symplectic Euler Scheme
+* Forward Euler Scheme
+* Taylor scheme
+* Verlet Velocity Scheme
+* Newmark Beta Method Scheme
+
+### Contact search
+
+The contact detection basically consists in determining, for every target object, what other objects are in contact with, and then, judge for the correspondent interaction. It is usually not needed to perform a search every time step, which is generally limited for the stability of the explicit integration of the equations of motion.
+A bins based technique is used for this purpose.
+
+## Available Interfaces
+
+### G-DEMPack
+
+G-DEMPack (formerly D-DEMpack) is the package that allows a user to create, run and analyze results of a DEM simulation for discontinuum / granular / little-cohesive materials. Requires [GiD](https://www.gidhome.com/) - Pre and Post Processing software.
+
+You can read the G-DEMPack manual or follow the G-DEMPack Tutorials included in DEMPack Tutorials for fast learning on how to use the GUI.
+
+### C-DEMPack
+
+C-DEMPack combines the features of G-DEMPack with the simulation of continuum/cohesive materials. Refer to the C-DEMPack manual or follow the C-DEMPack Tutorials included in DEMPack Tutorials for more information.
+
+### F-DEMPack
+
+F-DEMPack allows you to simulate a wide spectrum of problems involving the interaction of granular DEM and fluid.
+
+* **Miguel Angel Celigueta** - *Core Development* - [maceli@cimne.upc.edu](mailto:maceli@cimne.upc.edu)
+* **Salva Latorre** - *Granular materials* - [latorre@cimne.upc.edu](mailto:latorre@cimne.upc.edu)
+* **Ferran Arrufat** - *Cohesive materials* - [farrufat@cimne.upc.edu](mailto:farrufat@cimne.upc.edu)
+* **Guillermo Casas** - *Fluid coupling* - [gcasas@cimne.upc.edu](mailto:gcasas@cimne.upc.edu)
+* **Joaquín Irazabal** - *Particle clusters & DEM-Solid interaction* - [jirazabal@cimne.upc.edu](mailto:jirazabal@cimne.upc.edu)
+
+See also the list of [Kratos Team members](http://kratos-wiki.cimne.upc.edu/index.php/Who_are_we).
```

Comment:
I don't think that list is updated anymore. Maybe we can point them to the Teams

Reply:
I think it is good for these reasons:
1- The email is there (nobody can send you a private email if you only appear in the Team section)
2- The specific topic for every developer is written
3- We should edit this file quite often: every time we add or remove a feature, so we can change this list as well...

Reply:
I see, let's leave it like that, but we probably should create an updated version of that page in the new wiki ASAP so at least people there is still in CIMNE. My concern is that there are people listed like Nelson or Enric Bonet (to name a couple) that should not be contacted anymore. At least they should not be the first pick anymore.

For example Enric appears first in the list under the CSIC team so I would be tempted to send him a mail

Reply:
I was thinking... Having some names as reference contact people for each application may be a good idea but, rather than mails we could suggest opening an issue and mentioning the DEM team (or whichever is appropriate) as a contact point.

Reply:
@roigcarlo , I didn't understand you were talking about the last line (the reference to the old wiki). Now I understood. It has been removed.
@jcotela: concerning your comment, I will follow the general policy. However, I think the emails open the door to non-issue messages like 'would like to write a paper with us?' or 'can we do a research project together?'. General stuff not related to the code.